chicago/models.py

```diff
@@ -168,3 +175,103 @@ def most_recent_past_city_council_meeting(cls):
             )
         else:
             return None
+
+
+class ChicagoPerson(Person):
+    class Meta:
+        proxy = True
+        app_label = "chicago"
+
+    # this is a very expensive query - do not use on listing pages
+    @property
+    def full_attendance(self):
```

Comment:
moved this logic from the view into a new `ChicagoPerson` abstract model

Reply:
i'd make this a cached property